packages/chat-stateful-client/src/TestHelpers.ts

```diff
@@ -91,7 +91,7 @@ export type ChatClientWithEventTrigger = ChatClient & {
  * @private
  */
 export function createMockChatClient(): ChatClientWithEventTrigger {
-  const mockEventHandlersRef = { value: {} };
+  const mockEventHandlersRef: { value: any } = { value: {} };
```

Comment:
This was a quick change to enable the test code to pass; we now have `noImplicitAny` across the chat-stateful-client packlet.
Can look into a proper typing with the rest of the `any` removal.